topup.go

```diff
@@ -0,0 +1,51 @@
+package stripe
+
+type TopupParams struct {
+	Params    `form:"*"`
+	Amount    uint64        `form:"amount"`
+	Currency  Currency      `form:"currency"`
+	Desc      string        `form:"description"`
+	Source    *SourceParams `form:"*"` // SourceParams has custom encoding so brought to top level with "*"
+	Statement string        `form:"statement_descriptor"`
+}
+
+// SetSource adds valid sources to a TopupParams object,
+// returning an error for unsupported sources.
+func (p *TopupParams) SetSource(sp interface{}) error {
+	source, err := SourceParamsFor(sp)
+	p.Source = source
+	return err
+}
+
+// TopupListParams is the set of parameters that can be used when listing topups.
+// For more details see https://stripe.com/docs/api#list_topups.
+type TopupListParams struct {
+	ListParams   `form:"*"`
+	Created      int64             `form:"created"`
+	CreatedRange *RangeQueryParams `form:"created"`
+}
+
+// TopupList is a list of charges as retrieved from a list endpoint.
```

Comment:
Is `charges` an intentional word here, vs using `topups`?

Similarly for other references to `charge` in the description?